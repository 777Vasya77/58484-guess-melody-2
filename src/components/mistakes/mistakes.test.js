import React from 'react';
import renderer from 'react-test-renderer';
import Mistakes from '~/components/mistakes/mistakes';

describe(`Mistakes component tests`, () => {
  it(`Component render correctly`, () => {
    const tree = renderer
        .create(
            <Mistakes mistakes={3}/>
        ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
