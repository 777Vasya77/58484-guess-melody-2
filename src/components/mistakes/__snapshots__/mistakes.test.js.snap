// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mistakes component tests Component render correctly 1`] = `
<div
  className="game__mistakes"
>
  <div
    className="wrong"
  />
  <div
    className="wrong"
  />
  <div
    className="wrong"
  />
</div>
`;
